indra/newview/skins/default/xui/en/notifications.xml

```diff
@@ -413,6 +413,14 @@ Initialization with the Marketplace failed because of a system or network error.
         yestext="OK"/>
     </notification>
 
+    <notification
+     icon="alertmodal.tga"
+     name="InvalidKeystroke"
+     type="alertmodal">
+        There was an invalid keystroke entered. Please input a valid text.
```

Comment:
This message is too generic. If it doesn't provide a list of allowed symbols (declare and get a string from TypeValuesHelper?) it's better to replace it with a failure sound.